common/src/hardforks.rs

```diff
@@ -0,0 +1,54 @@
+use chrono::NaiveDateTime;
+
+use crate::network::Network;
+
+
+pub struct HardForks {
+    network: Network
+}
+
+// this function blindly unwraps only call when you know the timestamp is good
```

Comment:
Rust convention is to format satefy comment :
`// SAFETY: Reason`
maybe just change this comment as 
```
// This function use unwrap
// SAFETY : Its caller responsability to ensure the timestamp is valid.
```